src/spikeinterface/core/base.py

```diff
@@ -846,43 +846,63 @@ def save_to_memory(self, sharedmem=True, **save_kwargs) -> "BaseExtractor":
         return cached
 
     # TODO rename to saveto_binary_folder
-    def save_to_folder(self, name=None, folder=None, overwrite=False, verbose=True, **save_kwargs):
+    def save_to_folder(
+        self,
+        name: str = None,
+        folder: str | Path = None,
+        overwrite: str = False,
+        verbose: bool = True,
+        **save_kwargs,
+    ):
         """
-        Save extractor to folder.
+        Save the extractor and its data to a folder.
 
-        The save consist of:
-          * extracting traces by calling get_trace() method in chunks
-          * saving data into file (memmap with BinaryRecordingExtractor)
-          * dumping to json/pickle the original extractor for provenance
-          * dumping to json/pickle the cached extractor (memmap with BinaryRecordingExtractor)
+        This method extracts trace data, saves it to a file (using a memory-mapped format
+        with BinaryRecordingExtractor), and stores both the original extractor's provenance
+        and the cached extractor's metadata in JSON format.
```

Comment:
```suggestion
        and the cached extractor's metadata in JSON format.
```

Is pickle no longer possible at all?

Reply:
Didn't mean this one as a suggestion. Oops!

Reply:
Not for this function I think. Looking a the body of the function it does not seem like it is an option.

Reply:
Cool sorry I didn't look at the function I was just looking at the diff! We can skip this one.

Reply:
Good.